src/libs/elektra/keyset.c

```diff
@@ -1113,39 +1113,39 @@ KeySet * ksCut (KeySet * ks, const Key * cutpoint)
 			switch (ns)
 			{
 			case KEY_NS_SPEC:
-				strncpy (newname + 2, "spec", 4);
+				strncpy (newname + 2, "spec", 5);
```

Comment:
We could introduce a convenience macro ELEKTRA_COPY_CONSTANT_STRING which does strncpy(ptr, str, sizeof(str)).